Clients/Xamarin.Interactive.Client/Telemetry/Client.cs

```diff
@@ -116,12 +139,6 @@ public void Post (ITelemetryEvent evnt)
 
         async Task PostEventOnceAsync (ITelemetryEvent evnt)
         {
-            if (BuildInfo.IsLocalDebugBuild) {
```

Comment:
So now we're posting telemetry for local debug builds even if there isn't a local telemetry server running?

Reply:
No. If we don't have a valid endpoint URI, we won't have an `httpClient` instance, and the `Post` method below silently bails if so.